src/pages/office/cat-cpc/office.cat-cpc.page.ts

```diff
@@ -0,0 +1,384 @@
+        select(getCandidateDescription),
+      ),
+      displayIdentification$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getIdentification))),
+        map(([outcome, identification]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'identification', identification)),
+      ),
+      identification$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getIdentification),
+      ),
+      combination$: currentTest$.pipe(
+        select(getTestData),
+        select(getCombination),
+      ),
+      displayAdditionalInformation$: currentTest$.pipe(
+        select(getTestOutcome),
+        withLatestFrom(currentTest$.pipe(
+          select(getTestSummary),
+          select(getAdditionalInformation))),
+        map(([outcome, additional]) =>
+          this.outcomeBehaviourProvider.isVisible(outcome, 'additionalInformation', additional)),
+      ),
+      additionalInformation$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getAdditionalInformation),
+      ),
+      assessmentReport$: currentTest$.pipe(
+        select(getTestSummary),
+        select(getAssessmentReport),
+      ),
+      overallScore$: currentTest$.pipe(
+        select(getTestData),
+        select(getTotalPercent),
+      ),
+      question1$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion1),
+      ),
+      question2$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion2),
+      ),
+      question3$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion3),
+      ),
+      question4$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion4),
+      ),
+      question5$: currentTest$.pipe(
+        select(getTestData),
+        select(getQuestion5),
+      ),
+      testResult$: currentTest$.pipe(
+        select(getTestOutcomeDebrief),
+      ),
+      isRekey$: currentTest$.pipe(
+        select(getRekeyIndicator),
+        select(isRekey),
+      ),
+    };
+
+    const { testResult$, combination$ } = this.pageState;
+
+    this.subscription = merge(
+      testResult$.pipe(map(result => this.outcome = result)),
+      combination$.pipe(map(result => this.combinationAdditionalText = result)),
+    ).subscribe();
+
+    console.log('this.outcome', this.outcome);
+  }
+
+  popToRoot() {
+    const journalPage = this.navController.getViews().find(view => view.id === JOURNAL_PAGE);
+    this.navController.popTo(journalPage);
+  }
+
+  defer() {
+    this.popToRoot();
+    this.store$.dispatch(new SavingWriteUpForLater());
+    this.store$.dispatch(new PersistTests());
+  }
+
+  getCombinationAdditionalText(code): string {
+    const question: Combination = questionCombinations.find((question) => {
+      return question.code === code;
+    });
+
+    return question.additionalText;
+  }
```

Comment:
getCombinationAdditionalText(code: CombinationCodes): string {
    const question: Combination = questionCombinations.find((combination: Combination) => {
      return combination.code === code;
    });

    return question.additionalText;
  }

Reply:
was having a hard time with that....so obvious.  had to make a few other changes to accommodate it. Done.